frontend/add-client/confirm.component.tsx

```diff
@@ -20,7 +20,7 @@ export default function Confirm(props: StepComponentProps) {
           <button
             type="button"
             className="secondary"
-            onClick={() => props.goBack(Step.CLIENT_SOURCE)}
+            onClick={() => props.goBack(Step.SERVICES)}
```

Comment:
Fixed a random bug where the Confirm page would skip the SERVICES page when you pressed the Back button